src/query/storages/common/index/src/filters/xor8.rs

```diff
@@ -128,9 +128,16 @@ impl Filter for Xor8Filter {
 
 impl SupportedType for Xor8Filter {
     fn is_supported_type(data_type: &DataType) -> bool {
-        // Bloom index only enabled for String and Integral types for now
         let inner_type = data_type.remove_nullable();
-        matches!(inner_type, DataType::Number(_) | DataType::String)
+        matches!(
+            inner_type,
+            DataType::Number(_)
+                | DataType::String
```

Comment:
We used to support these types, but there were some performance issues(I can't find the PR ), so we removed them? cc @dantengsky

Reply:
why do we have performance issue about `timestamp,date` ?

Reply:
Confirmed, it's my mistake. We only support `string` data type, please ignore my comments.

Reply:
can not recall the details :(,  but it is not about performance (no benchmark of different data types has been done)